components/list/bullet/src/index.scss

```diff
@@ -46,6 +46,11 @@ $sz-list-bullet-illustration-l: 72px !default;
   }
 
   &-illustration {
+    .sui-ListBullet--responsive & {
+      @include media-breakpoint-up(l) {
```

Comment:
why not including it there?
https://github.com/SUI-Components/schibsted-spain-components/blob/master/components/list/bullet/src/index.scss#L32

Reply:
so you don't need the mq and the selector because they are already there